integration/tests/cook/test_basic.py

```diff
@@ -54,7 +54,7 @@ def test_no_cook_executor_on_subsequent_instances(self):
             pass
         job = util.load_job(self.cook_url, job_uuid)
         message = json.dumps(job, sort_keys=True)
-        job_instances = sorted(job['instances'], key=lambda instance: instance['end_time'])
```

Comment:
Could you figure out why `end_time` was missing? If the job was completed I'd expect it to be on the instance.

Reply:
As we discussed offline, the job wasn't completed.